c/planarityTestGraphFunctionality.c

```diff
@@ -109,8 +129,8 @@ int TestGraphFunctionality(char *commandString, char *infileName, char *inputStr
 						if (outputFileP == NULL)
 						{
 							charsAvailForFilename = (int) (MAXLINE - strlen(outfileName));
-							errorMessageFormatStr = "Unable to open file \"%.*s\" for output.\n";
-							sprintf(messageContents, errorMessageFormatStr, charsAvailForFilename, outfileName);
+							messageFormat = "Unable to open file \"%.*s\" for output.\n";
```

Comment:
starting with the `strOrFileP testOuptut` declaration and on down, all output code should be pull out of this function and put into a separate function that outputs the results of test all graphs. It should receive the stats object and the name of the output file, and do all the rest of the work related to outputting, testing for failures related to outputting, and returning NOTOK related to said failures, or OK on success.